lib/simplifyExpression/stepThrough.js

```diff
@@ -45,6 +46,10 @@ function stepThrough(node, debug=false) {
       logSteps(nodeStatus);
     }
     steps.push(removeUnnecessaryParensInStep(nodeStatus));
+    // Stop on division by zero
+    if (nodeStatus.changeType === ChangeTypes.DIVISION_BY_ZERO) {
```

Comment:
this would maybe come later, but eventually this will be a check to see if the changeType is an error

(oh, maybe a nodeStatus should have both a changeType and an errorType (default to null)? idk, still trying to figure out what this would look like long term)